components/brave_vpn/brave_vpn_service_desktop.h

```diff
@@ -53,6 +49,8 @@ class BraveVpnServiceDesktop
   void RemoveVPNConnnection();
 
   void AddObserver(Observer* observer);
+  void AddObserver(mojo::PendingRemote<brave_vpn::mojom::ServiceObserver>
```

Comment:
nit: it would be good to put together with other overridden methods at L61 below.